source/connectors/kafka-connect-migration.txt

```diff
@@ -0,0 +1,59 @@
+.. _kafka-connect-migration:
+
+==========================
+Migrate from Kafka Connect
+==========================
+
+.. default-domain:: mongodb
+
+This section lists the changes needed to migrate from the deprecated,
+community-developed version of the Kafka Connector (hosted in the
+`hpgrahsl/kafka-connect-mongodb GitHub repository
+<https://github.com/hpgrahsl/kafka-connect-mongodb>`_) to the `official
+connector <https://github.com/mongodb/mongo-kafka>`_.
+
+Once your project build file is updated to reference the official Kafka
+Connect library, use the following information to migrate your
+project.
```

Comment:
This seems like an important step. How can I find info on what to change in my build file?

Reply:
Would it be sufficient to reword this to say:

"Update any project dependencies that reference hpgrahsl/kafka-connect-mongodb to point to mongodb/mongo-kafka instead"
since there are several different tools the developer could be using such as Gradle, Maven, Ant, or other?